### ISSUE TITLE
Project milestone 1 review

### ISSUE THREAD

**Issue by hardikrakholiya** [airavata-courses/airavata-nextcoud#23]
Can you please email me all the required credentials?

**Comment by karankotz** [comment]
@hardikrakholiya please comment your email-id ?

**Comment by pkmsoftpro** [comment]
prkumoda@iu.edu

**Comment by hardikrakholiya** [comment]
hrakholi@iu.edu is my email. I have created an account in seagrid but its pending approval.

**Comment by hardikrakholiya** [comment]
Specifically I want to see how jenkins is setup for your repository and need credentials for that.

**Comment by karankotz** [comment]
@jitendraselvam Thanks !

**Comment by hardikrakholiya** [comment]
Thanks for sharing the credentials @karankotz . below are my comments for your project.

I think it would be better to put the project ideation in the home page of wiki since its empty and put the link inside readme.md using link format for markdown files. eg. [link to wiki](https://github.com/airavata-courses/airavata-nextcoud/wiki). But its a minor thing.

Overall, project Ideation page looks very professional. I will have my project idea page documented like that in the future. It nicely puts user stories, a simple but just about right napkin drawing and the value proposition.

I executed the build steps. I already had Apache Maven and Java setup in my system. So it was little easier for me to set up. Executing the command works as intended and builds the project.

I would like to give bonus points for making the demo video since that was out of scope for this assignment. 

I logged in to Nextcloud using the credentials provided. I really liked the UI. It is neat and reminds me of older Dropbox UI.

I logged in to Jenkins using the credentials provided. I verified that it executes the maven build for every commit.
